lib/batch_api/response.rb

```diff
@@ -32,6 +32,8 @@ def process_body(body_pieces)
       # (I'm looking at you, ActionDispatch::Response)
       # so turn it into a string
       body_pieces.join
+    ensure
+      body_pieces.close if body_pieces.respond_to?(:close)
```

Comment:
Not sure I understand the route that's supposed to happen. Is it enough that `body_pieces` has a call to `close`, despite the closed version not being returned at all?